ertviz/controllers/response_controller.py

```diff
@@ -51,33 +92,35 @@ def _get_observation_data(observation, x_axis):
         y_axis=[d + std for d, std in zip(data, stds)],
         text="Observations std upper",
         name="Observations std upper",
-        mode="line",
+        mode="lines",
         line=dict(color="red", dash="dash"),
         marker=None,
     )
     return [observation_data, lower_std_data, upper_std_data]
 
 
-def _create_response_model(url):
-    response = get_response(url)
+def _create_response_model(response, plot_type, selected_realizations):
```

Comment:
Can we rename this to `_create_response_plot` and maybe rename EnsemblePlotModel to ResponsePlotModel? Maybe we could have a little brainstorm on naming